module/x/gravity/keeper/signer_set_test.go

```diff
@@ -18,28 +18,20 @@ func TestSignerSetTxExecuted(t *testing.T) {
 		ctx,
 	)
 
-	input.GravityKeeper.CreateSignerSetTx(
-		ctx,
-	)
-
-	sstx1 := input.GravityKeeper.GetOutgoingTx(ctx, types.MakeSignerSetTxKey(1)).(*types.SignerSetTx)
-	sstx2 := input.GravityKeeper.GetOutgoingTx(ctx, types.MakeSignerSetTxKey(2)).(*types.SignerSetTx)
+	sstx := input.GravityKeeper.GetOutgoingTx(ctx, types.MakeSignerSetTxKey(1)).(*types.SignerSetTx)
```

Comment:
Why remove the testing of multiple pending TXs?

Reply:
It was really testing that one gets deleted by the `signerSetExecuted` function, which is functionality that we removed.

Reply:
Ah, and we have test coverage for the pruning then?